pkgs/tools/networking/dnsmasq/default.nix

```diff
@@ -19,6 +19,10 @@ stdenv.mkDerivation rec {
     sha256 = "1yzq6anwgr5rlnwydpszb51cyhp2vjq29b24ck19flbwac1sk73l";
   };
 
+  patches = [
+    ./makefile-defaults.patch
```

Comment:
Could you also post a link to mailing list archive if available. That makes
it easier to track the upstream status. Thanks!

Reply:
will do as soon as available

Reply:
That's still missing the link to the ML archive, or given there hasn't been too much positive reaction, we might probably just want to fall back to set this in `makeFlags`.

Reply:
Yes let's just do make flags, but still link the mailing list.

Reply:
i might be able to take care of this in the coming days. but can't promise as I'm currently very busy.